test_flake8_graphql.py

```diff
@@ -0,0 +1,28 @@
+# -*- encoding:utf-8 -*-
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+
+def test_GQL100_pass_1(flake8dir):
+    flake8dir.make_example_py("""
+        def gql(query):
+            return query
+
+
+        query = gql("query { countries { name } }")
+    """)
+    result = flake8dir.run_flake8()
+    assert result.out_lines == []
+
+
+def test_GQL100_fail_1(flake8dir):
+    flake8dir.make_example_py("""
+        def gql(query):
+            return query
+
+
+        query = gql("queryd countries { name } }")
+    """)
+    result = flake8dir.run_flake8()
+    assert result.out_lines[0] == (
```

Comment:
better as `assert result.out_lines == []` and asserting the whole list, checking you aren't raising any extra codes